samples/bluetooth/llpm/src/main.c

```diff
@@ -238,14 +239,32 @@ static int enable_llpm_mode(void)
 static int enable_llpm_short_connection_interval(void)
 {
 	int err;
-	struct bt_le_conn_param conn_param = {
-		.interval_min = INTERVAL_LLPM,
-		.interval_max = INTERVAL_LLPM,
-		.latency = 0,
-		.timeout = 300
-	};
+	struct net_buf *buf;
+
+	hci_vs_cmd_conn_update_t *cmd_conn_update;
+
+	buf = bt_hci_cmd_create(HCI_VS_OPCODE_CMD_CONN_UPDATE,
+				sizeof(*cmd_conn_update));
+	if (!buf) {
+		printk("Could not allocate command buffer\n");
+		return -ENOMEM;
+	}
 
-	err = bt_conn_le_param_update(default_conn, &conn_param);
+	u16_t conn_handle;
+	err = bt_hci_get_conn_handle(default_conn, &conn_handle);
+	if (err) {
+		printk("Failed obtaining conn_handle (err %d)\n", err);
+		return err;
+	}
+
+	cmd_conn_update = net_buf_add(buf, sizeof(*cmd_conn_update));
```

Comment:
Maybe we should find a common place for sending this command? Create a small LLPM abstraction layer.

Reply:
Under `subsys/bluetooth/llpm/llpm.c`?

Reply:
Possibly. Or maybe some collect similar smaller features in some common place for vendor specific HCI commands.

Reply:
It will require some more thinking and prototyping. Should we leave it out for now?

Reply:
Fine by me.